### DISCUSSION TITLE
Should I translate the Scratch Addons name?

### DISCUSSION THREAD

**Discussion by superscratch444tre** [ScratchAddons/ScratchAddons#1664] [Translation] [answered]
Should I translate the "addon" part? Or just let it ScratchAddons?

**Comment by cobaltt7** [comment]
I think it should be "Scratch Addons" for I'm not completely sure
________________________________

**Comment by GrahamSH-LLK** [comment]
Scratch Addons should be translated as "Scratch Addons".

**Comment by apple502j** [comment, marked as answer]
Scratch Addons can be translated (if translating it makes more sense than calling it "Scratch Addons" - good example would be Chinese)

**Reply by WorldLanguages** [reply]
Yeah, use common sense. Google has a similar dilemma with their product names - I know many non-English languages still call Google Maps by its English name, for example.
